Competition2018/src/main/java/competition/subsystems/elevator/commands/CalibrateElevatorViaStallCommand.java

```diff
@@ -5,25 +5,32 @@
 import competition.subsystems.elevator.ElevatorSubsystem;
 import edu.wpi.first.wpilibj.Timer;
 import xbot.common.command.BaseCommand;
+import xbot.common.controls.sensors.TalonCurrentMonitor;
 import xbot.common.injection.wpi_factories.CommonLibFactory;
 import xbot.common.properties.DoubleProperty;
 import xbot.common.properties.XPropertyManager;
 
 public class CalibrateElevatorViaStallCommand extends BaseCommand {
     ElevatorSubsystem elevator;
     CommonLibFactory clf;
+    TalonCurrentMonitor currentMonitor;
 
     DoubleProperty power;
     DoubleProperty calibrationTime; // In milliseconds
+    DoubleProperty calibrationCurrentThreshold;
+    DoubleProperty goodCurrentRange;
 
     double targetTime;
     boolean atTarget;
 
+
     @Inject
     public CalibrateElevatorViaStallCommand(XPropertyManager propMan, CommonLibFactory clf) {
         this.clf = clf;
         power = propMan.createPersistentProperty("Elevator Jamming Calibration Power", -0.2);
         calibrationTime = propMan.createPersistentProperty("Elevator Calibration time (ms)", 4000);
+        calibrationCurrentThreshold = propMan.createPersistentProperty("Calibration Current Threshold", 15);
+        this.requires(elevator);
```

Comment:
You can't require the elevator, as this would stop other elevator commands from running.

Reply:
Also, you need to initialize your TalonCurrentMonitor